docs/quickstart.rst

```diff
@@ -23,6 +23,16 @@ Fairlearn is also available on
 For checking out the latest version in our repository check out our
 :ref:`advanced_install`.
 
+.. note::
+
+    The Fairlearn API is still evolving, so example code in 
+    this documentation may not work with every version of Fairlearn.
+    Please use the version selector to get to the instructions for
+    the appropriate version (unfortunately there are no instructions
```

Comment:
i would just remove the parenthesis starting with `(unfortunately ...`. it's too confusing.

Reply:
Done